catroid/src/org/catrobat/catroid/content/bricks/WhenStartedBrick.java

```diff
@@ -43,6 +43,10 @@
 
 	public WhenStartedBrick(Script script) {
 		this.script = script;
+
+		if (script != null && script.isCommentedOut()) {
+			setCommentedOut(true);
+		}
 	}
```

Comment:
Why is the clone method not altered in order to set the cloned brick comment state ?